### PR TITLE
Remove spring-milestone repository

### PR DESCRIPTION
**Pull request by moarychan**
# Description

- Bump the version `spring-cloud-azure-dependencies` to 6.0.0-beta.5.
- Remove `spring-milestone` repository.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.azure.spring</groupId>
   <artifactId>spring-cloud-azure-dependencies</artifactId>
-  <version>6.0.0-beta.4</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-dependencies;current} -->
+  <version>6.0.0-beta.5</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-dependencies;current} -->
```
Comment (backwind1233) [inline sdk/boms/spring-cloud-azure-dependencies/pom.xml:9]: Does version_client.txt need change?

**Comment by backwind1233** [comment]
Should we care about this error?
image

File: sdk/boms/spring-cloud-azure-dependencies/pom.xml (hunk 1 above)
Comment (moarychan) [reply sdk/boms/spring-cloud-azure-dependencies/pom.xml:9]: Updated.

**Comment by moarychan** [comment]
> Should we care about this error? image

Yes, it's normal for the Spring Cloud Azure side.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**